happi/tests/test_cli.py

```diff
@@ -618,6 +616,46 @@ def test_load(
         assert "Creating shell with devices" in caplog.text
 
 
+def test_load_multiple_args(
+    caplog: pytest.LogCaptureFixture,
+    client: happi.client.Client,
+    happi_cfg: str,
+    runner: CliRunner
+):
+    # try to load the item
+    devices = {}
+    devices['tst_base_pim'] = client.load_device(name='tst_base_pim')
+    devices['tst_base_pim2'] = client.load_device(name='tst_base_pim2')
+    with mock.patch.object(IPython, 'start_ipython') as m:
+        _ = runner.invoke(
+            happi_cli, ['--path', happi_cfg, 'load', 'tst_base_pim', 'tst_base_pim2']
+        )
+        m.assert_called_once_with(argv=['--quick'], user_ns=devices)
+    with caplog.at_level(logging.INFO):
+        assert "Creating shell with devices" in caplog.text
+
+
+def test_load_glob_args(
+    caplog: pytest.LogCaptureFixture,
+    client: happi.client.Client,
+    happi_cfg: str,
+    runner: CliRunner
+):
+    # try to load the item
+    results = client.search_regex(name='tst_base_.*')
+
+    devices = {}
+    devices['tst_base_.*'] = [res.item['name'] for res in results]
+
+    with mock.patch.object(IPython, 'start_ipython') as m:
+        _ = runner.invoke(
+            happi_cli, ['--path', happi_cfg, 'load', 'tst_base_.*']
```

Comment:
This doesn't pick up any results because you use glob-style matching by default, which does not interpret `.` as a wildcard character.  You either have to set `--use_glob=False` or remove the `.`

Reply:
You then also have to match your search call with this invocation.  (you explicitly search-regex at the beginning of this test)